store/getters.js

```diff
@@ -276,7 +276,9 @@ export const families = (state, getters) => {
               !isEmpty(sample.comment)),
           sexValid: constants.sexOptions.includes(sample.sex),
           sourceValid:
-            sample.internalId || getters.orderSources.includes(sample.source),
+            (sample.internalId ||
+              getters.orderSources.includes(sample.source)) &&
+            hasFFPEandWGtag(sample),
```

Comment:
```suggestion
            isNotFFPEandWGtag(sample),
```
seems like the name is reversed to the logic in the function, right?